.github/workflows/update-database.yml

```diff
@@ -60,6 +62,21 @@ jobs:
         working-directory: repo
         run: brew which-update --stats executables.txt
 
+      - name: Install oras for pushing to GitHub Packages
+        if: github.ref == 'refs/heads/main'
+        run: brew install oras
+
+      - name: Log in to GitHub Packages
+        if: github.ref == 'refs/heads/main'
+        run: echo "${{ secrets.GITHUB_TOKEN }}" | oras login ghcr.io --username brewtestbot --password-stdin
```

Comment:
can handle in follow-up but: nice to test such things on every commit when possible.